en/asset/label-atlas.md

```diff
@@ -0,0 +1,32 @@
+# LabelAtlas Asset
+
+**LabelAtlas** asset is a user defined asset, it's used for configuring LabelAtlas.
+
+## Create LabelAtlas Asset
+
+In **Assets Panel**  right-click on a folder, and click the following context menu item **Create -> Label Atlas**. It will create a **LabelAtlas.labelatlas** asset.
+
+![create label atlas](label-atlas/create-label-atlas.png)
+
+Before using **LabelAtlas** asset, you need some configurations. eg, configure a pre-drawn picture that contains the font style, as shown below:
```

Comment:
Before using **LabelAtlas** asset, you need some configurations. eg, configure a pre-drawn picture that contains the font style, as shown below:
-->
Before using the **LabelAtlas** asset, it needs some configuration. Configure a pre-drawn picture that contains the font style, as shown below: